### DISCUSSION TITLE
How can I change git related defaul icons?

### DISCUSSION THREAD

**Discussion by dehghani-mehdi** [JanDeDobbeleer/oh-my-posh#2698] [Troubleshoot]
I could not find any related issue/article/ ... about it, so I asked here.
For example: 
![Untitled](https://user-images.githubusercontent.com/9152330/186181290-f589e7ba-9f75-4746-9b7a-961f7e0534ec.png)

**Comment by AjayKMehta** [comment]
Check the docs:

https://ohmyposh.dev/docs/segments/git

**Reply by dehghani-mehdi** [reply]
I already mentioned in my comment, I didn't find any related/useful info about it, if you know something, please share

**Reply by JanDeDobbeleer** [reply]
@dehghani-mehdi those are in the template, not the icon set as they're a single value without logic (unlike branch icon for example).

**Reply by dehghani-mehdi** [reply]
@JanDeDobbeleer so those are unchangeable?

**Reply by JanDeDobbeleer** [reply]
@dehghani-mehdi you can adjust the template, they're in there. You can find the default template in the [docs](https://ohmyposh.dev/docs/segments/git#template-info) (as well as these icons in there).

**Reply by dehghani-mehdi** [reply]
Can I change the icon of `{{ .BranchStatus }}` too?

**Reply by JanDeDobbeleer** [reply]
@dehghani-mehdi `.BranchStatus` uses the [branch icons](https://ohmyposh.dev/docs/segments/git#branch) based on the state.

**Reply by dehghani-mehdi** [reply]
@JanDeDobbeleer thank you for your help and also thank you for `oh-my-posh`
